opal-ui/src/components/project/AddProjectDialog.vue

```diff
@@ -227,19 +221,29 @@ async function onAddProject() {
       editMode.value
         ? await projectsStore.updateProject(newProject.value as ProjectDto)
         : await projectsStore.addProject(newProject.value as ProjectDto),
-        onHide();
-        emit('update');
+        emit('update', newProject.value);
+      onHide();
     } catch (err) {
       notifyError(err);
     }
   }
 }
 
 onMounted(() =>
-  profilesStore.initProfile().then(() =>
-    filesStore.initFiles(`/home/${profile.value.principal}/export`).then(() => {
+  profilesStore.initProfile().then(() => {
+    filesStore.initFiles(`/home/${profile.value.principal}`).then(() => {
```

Comment:
This is supposed to be a folder that is linked to a project, so it should not be a specific user's folder. I would not put such default value, but how is it in the old UI?

Reply:
In the old UI, since the project still doesn't exist, the File Browser opens in the Home folder:
![image](https://github.com/user-attachments/assets/328dbac7-98b4-49df-8199-e7eed16805ef)